src/app/pages/quickMaps/components/sideNavContent/sideNavContent.component.html

```diff
@@ -9,92 +9,93 @@ <h2 (click)="quickMapsService.sideNavToggle()">Quick Maps</h2>
     class="bmgf-maps-nice-scrollbar scroll-area">
     <form [formGroup]="quickMapsForm" id="quickMapsForm"
         [ngClass]="(quickMapsService.slimObservable | async) ? 'slim-sidenav' : ''">
-        <div class="flex-row">
-            <div class="flex-row">
+
+        <fieldset name="geographyOfInterest">
+            <legend>
                 <h3>Geography of interest</h3>
                 <i class="fas fa-question-circle" matTooltip="some text exaplaining this form field"
                     style="color: #828282;"></i>
+                <a mat-raised-button [routerLink]="ROUTES.QUICK_MAPS_LOCATION_SELECT | route"
```

Comment:
This button doesn't sit right in the fieldset wrapper now as it's right on the border.

Reply:
Button should look better now